src/components/modules/blockSelection.ts

```diff
@@ -296,10 +296,10 @@ export default class BlockSelection extends Module {
 
     this.selectedBlocks.forEach((block) => {
       /**
-       * Make <p> tag that holds clean HTML
+       * Make <div> tag that holds clean HTML
        */
       const cleanHTML = clean(block.holder.innerHTML, this.sanitizerConfig);
-      const fragment = $.make('p');
+      const fragment = $.make('div');
```

Comment:
I think we better to check if cleanHTML is already a block tag (eg. h1-6 or ul/ol) and do not wrap it in this case.
But usage of `div` for paragraph would be semantically wrong I guess (and also seems would break pasting back)